src/io/flutter/run/daemon/DaemonConsoleView.java

```diff
@@ -46,6 +46,7 @@ protected ConsoleView createConsole() {
     };
 
     // Set up basic console filters. (More may be added later.)
+    // TODO(devoncarew): Do we need this filter? What about DartConsoleFilter (for package: uris)?
```

Comment:
I think you want DartConsoleFilter here, too, but it may be getting added at a later stage. I forgot just how convoluted this is.

Reply:
We may just need the DartConsoleFilter in the SdkRunConfig.java (the run/debug console)? If you're certain I can add it, but would otherwise hold off.